plugin-build/src/main/kotlin/io/sentry/android/gradle/autoinstall/AbstractInstallStrategy.kt

```diff
@@ -27,13 +29,27 @@ abstract class AbstractInstallStrategy : ComponentMetadataRule {
             }
             return
         }
-        val thirdPartySemVersion = SemVer.parse(context.details.id.version)
-        if (thirdPartySemVersion < minSupportedThirdPartyVersion) {
-            logger.warn {
-                "$sentryModuleId won't be installed because the current version is " +
-                    "lower than the minimum supported version ($minSupportedThirdPartyVersion)"
-            }
-            return
+        minSupportedThirdPartyVersion?.let {
+            parseVersion(context.details.id.version)?.let { thirdPartySemVersion ->
```

Comment:
`l`: we could also extract `parseVersion(context.details.id.version)` into a variable to avoid double-parsing the same thing

Reply:
my idea was to not even attempt parsing, if we don't care about the version anyways

Reply:
sounds good 👍